### PR TITLE
Removes external wounds

### PR DESCRIPTION
**Pull request by QualityVan**
## About The Pull Request
Wounds exist as a whole thing on top of limbs, tracking their individual damage and description and treatedness. The only gain from this is that if you examine someone you get to see what their wounds look like. This pr kills wounds as a datum with the exception of internal bleeding, which is refactored to process itself instead of being handled by the limb. Everything should work the same, with the sole exception of internal bleeding not contributing directly to the limb's brute loss.

## Why It's Good For The Game
Removes a level of abstraction which generally isn't thought about ingame and only really offers flavor (individual wound descs) as a benefit.

## Changelog
:cl:
refactor: Limbs have damage themselves instead of separating it into wounds
refactor: Internal bleeding doesn't count directly as damage on a limb.
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
What's the benefit of this?

File: code/modules/surgery/internal_bleeding.dm [hunk 1]
```diff
@@ -31,10 +30,8 @@
 	user.visible_message(span_notice("[user] has patched the damaged vein in [target]'s [affected.display_name] with \the [tool]."), \
 		span_notice("You have patched the damaged vein in [target]'s [affected.display_name] with \the [tool]."))
 
-	for(var/datum/wound/W in affected.wounds)
-		if(W.internal)
-			affected.wounds -= W
-			affected.update_damages()
+	for(var/datum/wound/internal_bleeding/W in affected.wounds)
+		qdel(W)
```
Comment (TiviPlus) [inline code/modules/surgery/internal_bleeding.dm:34]: Could you use QDEL_LIST() here

**Review by TiviPlus — APPROVED** [review]

**Comment by Al-1ce** [comment]
There is literally no point or benefit in this game. Why?

**Comment by QualityVan** [comment]
> There is literally no point or benefit in this change. Why?

Is there a reason you'd want wounds to keep existing as is?

**Review by BraveMole — APPROVED** [review]

**Comment by Al-1ce** [comment]
> Is there a reason you'd want wounds to keep existing as is?


It's harmless fluff. Additionally, it previously served as a quick "heads up" to any general marines who might need to tend an ally should a medic not be present. (As most marines do not carry scanners/or have valky)

There is literally no reason for this change. You listed removing "abstraction/flavor" as if we haven't seen tens if not hundreds of PRs similar in nature shut down for remvoing just that - harmless, if not somewhat helpful fluff.

You're the one under the burdon of reason, **not me**. You didn't respond or give a reason to the other person in this PR asking 'why', either. You should have a reason for why you'd want something merged, rather than demanding one from me. Needless to say, I'm not sure how this slipped by Mole who has denied similar PRs just like this. Fluff is exactly that - there for flavor. Might as well remove gun descriptions and char-bio's while you're at it.

**Comment by QualityVan** [comment]
> Additionally, it previously served as a quick "heads up" to any general marines who might need to tend an ally

So for this specifically, I actually kept that behavior of damage being visible on examine. It's still not usable most of the time because they only display if there's no clothing covering that limb, but it's there.

In terms of reason, that's in the pr body. It's an entire level of mechanics which add obscurity, being almost entirely invisible in game and then causing confusion when they actually matter for once (do you know how many people asked me why the bandage chain pr would target healed limbs? it was several). It's not that I want to remove it because it's flavor, but because it's detrimental to understanding and upkeep with the only benefits being flavor text that, again, is invisible 99% of the time because it only shows up on examine for undressed mobs and, if you consider it a benefit, another step in the limb infection chain.

**Pull request merged**